src/lib/packet.h

```diff
@@ -377,6 +377,7 @@ typedef enum {
                                * IETF-S/MIME) */
     PGP_PKA_EDDSA = 22,       /* EdDSA from draft-ietf-openpgp-rfc4880bis */
 
+    PGP_PKA_SM2_ENCRYPT = 98, /* SM2 encryption */
```

Comment:
SM2 has separate modes for DSA and Public Key Encryption, it might be clearer in implementation to not mix them up?